src/script/api/script_basestation.cpp

```diff
@@ -63,5 +63,5 @@
 {
 	if (!IsValidBaseStation(station_id)) return ScriptDate::DATE_INVALID;
 
-	return (ScriptDate::Date)::BaseStation::Get(station_id)->build_date;
+	return (ScriptDate::Date)(int32_t)::BaseStation::Get(station_id)->build_date;
```

Comment:
It feels weird to cast to an `int32_t` and then immediately to a `ScriptDate::Date`. Can we not do this in one step? (Honest question, I have no idea how ScriptDates work, nor casting in general. 😅)

Reply:
Sadly, no. The first case changes the StrongTypeDef to an integer, the second to the Script-specific type. It is a bit annoying, but it is what it is. Endless casts in scripts are just .. what they are :P